code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -73,6 +73,9 @@ Pipenet stuff; housekeeping
 			P.build_pipeline(src)
 
 /obj/machinery/atmospherics/components/proc/nullifyPipenet(datum/pipeline/reference)
+	if(!reference)
+		throw EXCEPTION("nullifyPipenet(null) called by [src.name] on [src.x] [src.y] [src.z]")
```

Comment:
Use CRASH instead.

Reply:
just adding on to this, you should separate the coords with commas

Reply:
Entire line should be CRASH("nullifyPipenet(null) called by [type] on [COORD(src)]")